tests/aggregation_regress/tests.py

```diff
@@ -1348,7 +1348,7 @@ def test_fobj_group_by(self):
 
     def test_annotate_reserved_word(self):
         """
-        Regression #18333 - Ensure annotated column name is properly quoted.
+        Regression #18333 - Annotated column name is properly quoted.
```

Comment:
Use ticket style described at https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style, e.g. "Annotated column name is properly quoted (#18333)."